### DISCUSSION TITLE
MSP SO Deployments

### DISCUSSION THREAD

**Discussion by ghostwalker42** [Security-Onion-Solutions/securityonion#8815] [General]
Hi All, 

I work for an MSP and we are looking for a Security tools.  I have come access SO and really like that its an all-in-one system.  The main question is that we want a central server for all the reporting and data. When deploying the system, We can deploy just the senors to each of our clients and then have the central server in our Data center.  would we need to VPN the Sensors back or is it encrypted and we can run them over the internet? 

sorry for the nub questions. I have been looking at a lot of different systems and or building one with Wahuz, MISP, TheHive, ECT... 

Thanks for the help in advance,

**Comment by weslambert** [comment]
Yes, traffic from the sensors to the manager is encrypted.
